packages/firebase-frameworks/README.md

```diff
@@ -0,0 +1,135 @@
+# Firebase CLI & Web Frameworks
+
+## Frameworks
+
+| Framework  | Support | |
+| ---------- | ------- | - |
+| [Next.js](https://firebase.google.com/docs/hosting/frameworks/nextjs) | **Early preview** | |
+| [Angular](https://firebase.google.com/docs/hosting/frameworks/angular) | **Early preview** | |
+| [Express](https://firebase.google.com/docs/hosting/frameworks/express) | **Early preview** | |
+| Flask | **Early preview** | Coming soon... |
+| Django | Experimental | Coming soon... |
+| [Flutter](https://firebase.google.com/docs/hosting/frameworks/flutter) | Experimental | |
+| Nuxt | Experimental | |
+| Astro | Experimental | |
+| SvelteKit | Experimental | |
+| Preact<br>React<br>Lit<br>Svelte<br>and more... | Experimental | Static web apps, powered by *Vite* |
```

Comment:
maybe replace <br> with commas? The <br> seems to break markdown's column width calculation.